pkg/machine/qemu/config.go

```diff
@@ -4,12 +4,24 @@
 package qemu
 
 import (
+	"errors"
+	"os"
 	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	Testing string = "testing"
+	Next    string = "next"
+	Stable  string = "stable"
```

Comment:
Could you add comments?

Probably worth adding some context to variable names. TestingStream?